Disable unnecessary namespace

sacarr opened the pull request:
@glennsl,  having now converted my own project's tests to rescript-jest, I think the Readme should mention that a namespace now exists that requires a change to bs-jest tests.  API calls have to be prefixed with the namespace, or code needs to be added to each test source to open the namespace.

``` rescript
open GlennslRescriptJest
```

glennsl commented:
Or we could just disable the namespace as its really rather pointless when it's already a single module with a reasonably unique name. What do you think?

sacarr commented:
I am fine with removing the namespace.  I added it to further differentiate rescript-jest from bs-jest but I did not realize Rescript would append the scope and the repo name.  I pushed a second commit that reverts the change to Readme and sets namespace to false in the bsconfig.json.

I tested with npm links to two of the repos I am working on where I had added the namespace to the tests.  I removed it all the tests passed without the namespace.

glennsl reviewed (approved):
Thanks! Yeah. the namespaces are a bit of a mouthful aren't they. There's also the possibility of assigning a custom name to the `namespace` property, but again I don't think that's necessary since it's already a single module and would just add more indirection.

glennsl commented:
Released as 0.9
